build.gradle

```diff
@@ -1,12 +1,12 @@
 plugins {
     id 'java'
-    id 'com.github.spotbugs' version '5.0.9'
+    id 'com.github.spotbugs' version '5.0.10'
```

Comment:
NOTE: From version 5.0.11 onward, spotbugs plugin becomes incompatible with Java 1.8. 
Either need to 
1. Update java engine for github workflow to a newer version of Java.
2. Keep the spot bugs version at 5.0.10 (until a fix is made)?

Reply:
looks are minor updates, no major/critical issue 
https://github.com/spotbugs/spotbugs-gradle-plugin/releases
I will say leave 5.0.10 for now